bcda/client/bluebutton.go

```diff
@@ -194,3 +205,12 @@ func GetDefaultParams() (params url.Values) {
 	params.Set("_format", "application/fhir+json")
 	return params
 }
+
+func HashHICN(toHash string) (hashedValue string) {
+	pepper, err := hex.DecodeString("b8ebdcc47fdd852b8b0201835c6273a9177806e84f2d9dc4f7ecaff08681e86d74195c6aef2db06d3d44c9d0b8f93c3e6c43d90724b605ac12585b9ab5ee9c3f00d5c0d284e6b8e49d502415c601c28930637b58fdca72476e31c22ad0f24ecd761020d6a4bcd471f0db421d21983c0def1b66a49a230f85f93097e9a9a8e0a4f4f0add775213cbf9ecfc1a6024cb021bd1ed5f4981a4498f294cca51d3939dfd9e6a1045350ddde7b6d791b4d3b884ee890d4c401ef97b46d1e57d40efe5737248dd0c4cec29c23c787231c4346cab9bb973f140a32abaa0a2bd5c0b91162f8d2a7c9d3347aafc76adbbd90ec5bfe617a3584e94bc31047e3bb6850477219a9")
```

Comment:
Should this be a constant?

Reply:
should we protect this salt value like a secret or is it okay like this?

Reply:
I firmly believe that this is a PUBLIC value that BB must share with anyone who is expected to request data from the hashedhicn endpoint.  Why do you think this should be protected?

Reply:
I agree with that.  Can we make it a constant so its easy to update if the need arises?

Reply:
> Should this be a constant?

This fundamentally can not ever change or else every hash in BB back end would need to be recomputed.  I can certainly move it, but I don't think it will get us that much.

Reply:
Not saying it "should or shouldn't" be protected, just thought I'd raise the question here.